README.md

```diff
@@ -121,50 +123,3 @@ module "cloud_security" {
 | <a name="output_security_hub_sns_topic"></a> [security\_hub\_sns\_topic](#output\_security\_hub\_sns\_topic) | The SNS topic that was created |
 | <a name="output_security_hub_sns_topic_subscriptions"></a> [security\_hub\_sns\_topic\_subscriptions](#output\_security\_hub\_sns\_topic\_subscriptions) | The SNS topic that was created |
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-
-### Git commits
```

Comment:
why this all stuff is removed from root file i.e. README.md?

cc: @tsaucier-sf  @jamescrowley321 @vijay-stephen

Reply:
Why is this content removed?

Reply:
reverted